neighsyncd/neighsync.cpp

```diff
@@ -15,6 +15,11 @@
 #include "warm_restart.h"
 #include <algorithm>
 
+#ifndef NTF_EXT_LEARNED
+/* from include/uapi/linux/neighbour.h */
+#  define NTF_EXT_LEARNED (1 << 4)
```

Comment:
Prefer to avoid such `#defines` in code. can you check?

Reply:
I just wasn't sure if the kernel headers were installed while building sonic-swss or not.  I just pushed a commit to include the header and see if it can build.